estate/models/estate_property.py

```diff
@@ -20,3 +20,12 @@ class EstateProperty(models.Model):
     garden_orientation = fields.Selection(
         string='Orientation',
         selection=[('North', 'North'), ('South', 'South'), ('East', 'East'), ('West', 'West')])
+    state = fields.Selection(
+        default='New',
+        selection=[
+            ('New', 'New'),
+            ('Offer Received', 'Offer Received'),
+            ('Offer Accepted', 'Offer Accepted'),
+            ('Sold', 'Sold'),
+            ('Canceled', 'Canceled')])
```

Comment:
It is better to use python like style names for selection as a convention, so that it would be convenient to not to think about exact user's text if it is with uppercase or not/space or underscore and etc.
```suggestion
        default='new',
        selection=[
            ('new', 'New'),
            ('offer_received', 'Offer Received'),
            ('offer_accepted', 'Offer Accepted'),
            ('sold', 'Sold'),
            ('canceled', 'Canceled')])
```